### ISSUE TITLE
JEM 2.1.4 problem import csv on empty fields

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#971]
earlier I imported the same files in 2.1.4 there were no such problems.
after updating to dev6, when I give JEM rights to all registered people,
I see only two categories, even when there are no groups and no groups assiciated to categories.
All 

and these two categories look like:

Taizé
? Firmpraktika                     (this is a sub-sub-category) from another not visible category

**Comment by Hoffi1** [comment]
Hm... that's hard to follow without details.
You say "import" so maybe it's incomplete, I mean possibly an access level is set which acutally doesn't exist? Maybe the level shown in backend isn't the level stored - Save should guarantee consistence.
Or it's something completely different...

**Comment by jojo12** [comment]
I changed the title because there seems to be the problem:
When I export the csv like I just did a lot of times and then import it in dev6:
events with only startdate have new a enddate later.
There must be something wrong, which is quite new
my import settings are always yes, even when there is no event before.

**Comment by Hoffi1** [comment]
You're right, there is a problem importing empty fields. That's because empty fields are skiped and so previous data is wrongly used.

Solution: Remove all data before binding new data.

**Comment by jojo12** [comment]
the date problem works now, thanks. 
For the other problem I create a new issue
